src/main/java/de/tum/in/test/api/ast/type/ExceptionHandlingType.java

```diff
@@ -0,0 +1,54 @@
+package de.tum.in.test.api.ast.type;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.ast.stmt.*;
+import com.github.javaparser.utils.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum ExceptionHandlingType {
+    /**
+     * Returns a list with all exception handling types
+     */
+    ALL(new ArrayList<>() {{
+        add(new Pair<>("Assert-Statement", AssertStmt.class));
```

Comment:
(But a use case or scenario would be interesting)

Reply:
A use case would be `ClassWithAnyKindsOfExceptionHandlings.java`, where we would like to check whether the method `assertStatement() `contains the unwanted node `assert` (in case we have an exercise where asserts are forbidden).